### PR TITLE
openstack: nginx root moved to /var/www/html

### PR DESCRIPTION
**Pull request by dachary**
In ubuntu 16.04

Signed-off-by: Loic Dachary <ldachary@redhat.com>

### CODE REVIEW

**Comment by dachary** [comment]
testing it now

File: teuthology/openstack/setup-openstack.sh [hunk 1]
```diff
@@ -230,6 +230,7 @@ function setup_pulpito() {
 
     sudo apt-get -qq install -y --force-yes nginx
     local nginx_conf=/etc/nginx/sites-available/default
+    sudo perl -pi -e 's|root /var/www/html|root /usr/share/nginx/html|' $nginx_conf
```
Comment (smithfarm) [inline teuthology/openstack/setup-openstack.sh]: so, in Ubuntu 16.04, nginx changed default root to /var/www/html but this was not reflected in /etc/nginx/sites-available/default ? Strange if that file is packaged with nginx

Comment (smithfarm) [inline teuthology/openstack/setup-openstack.sh]: oh, now I get it - you're changing the new default back to the old one

**Comment by dachary** [comment]
http://167.114.253.228/ubuntu-2017-02-08_23:25:14-dummy-master---basic-openstack/1/teuthology.log works

**Pull request merged**